.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] I strongly suggest adding node_modules directory in `.gitignore `file (as it can be added by accident while setting up Linters).